src/cirrus/plugins/management/commands/manage.py

```diff
@@ -111,6 +111,44 @@ def refresh(deployment, stackname=None, profile=None):
     deployment.refresh(stackname=stackname, profile=profile)
 
 
+@manage.command("run-workflow")
+@click.argument(
+    "payload-path",
+)
+@click.option(
+    "-t",
+    "--timeout",
+    type=int,
+    default=3600,
+    help="Maximum time (seconds) to allow for the workflow to complete",
+)
+@click.option(
+    "-f",
+    "--force-rerun",
+    is_flag=True,
+    help="Force workflow to run",
+)
+@click.option(
+    "-o",
+    "--out-path",
+    type=str,
+    default="",
+    help="Write output payload to the given path",
+)
```

Comment:
Looks like the payload is being dumped to stdout. Do you need an output file if you can just redirect stdout to a file?

Reply:
True.  That change is also inbound.